packages/cli/src/index.tsx

```diff
@@ -32,14 +32,14 @@ switch (command.name) {
 
   case PheroCommandName.Client:
     checkAndWarnForVersions([process.cwd()], console.warn)
-      .then(() => redirect("./node_modules/.bin/phero-client", command.argv))
+      .then(() => redirect("phero-client", command.argv))
```

Comment:
If we have `spawnPheroClient` and `spawnPheroServer` do we need a redirect?

Reply:
Yes, because they work a bit differently: While `redirect` is more a fire-and-forget, `spawnNpmExec` (which will become `spawnPheroClient`/`spawnPheroServer`) is expected to be a long-running process, especially for the dev-environment. This kills the app when the child-process gets killed for example, something that would never be expected in the dev-env. I will take the naming of the functions another look though 👍

Reply:
Now that I think about it a second time, a more concrete difference is that with these redirects the output is streamed directly to the shell. With `spawnNpmExec` we catch that output and make react render a custom UI from that data.